core/templates/domain/learner_dashboard/learner-playlist.service.ts

```diff
@@ -66,38 +67,18 @@ angular.module('oppia').factory('LearnerPlaylistService', [
         templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
           '/pages/learner-dashboard-page/modal-templates/' +
           'remove-activity-from-learner-dashboard-modal.template.html'),
-        backdrop: true,
```

Comment:
Please explain why you removed this. (Do this in general when you delete code that's not in a test file.)

Reply:
It wasn't my intention to delete it, I've fixed already!